frontend/src/pages/Q&A/Components/GetAnswer/GetAnswer.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect } from "react";
+import { useState } from "react";
+import "./GetAnswer.scss";
+import "../../../Q&A/Ques.scss";
+import { END_POINT } from "../../../../config/api";
+import { useParams } from "react-router-dom";
+
+function GetAnswer() {
+  const { answerId } = useParams();
+  // console.log(answerId);
+
+  const [answer, setAnswer] = useState([]);
+
+  //Get Answer
+  const GetAnswer = (answerId) => {
+    try {
+      fetch(`${END_POINT}/answers/${answerId}`, {
```

Comment:
Add page loader and generic toast message Which should show success message on success and failure message on failure.